intermediate_source/char_rnn_classification_tutorial.py

```diff
@@ -364,19 +364,19 @@ def timeSince(since):
 
 start = time.time()
 
-for epoch in range(1, n_epochs + 1):
```

Comment:
OK, I can change it to be that way.  Also, I was thinking to delete the comment that says you don't understand why initializing the loss to zero works since I don't think it is good to give people more things to feel confused about, and anyway I just assumed pytorch allows you to create a Variable by adding a python constant to a Variable, which doesn't seem mysterious, or am I missing something?